clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2705,6 +2705,30 @@ An error will be given if:
   }];
 }
 
+def AMDGPUNumWorkGroupsDocs : Documentation {
+  let Category = DocCatAMDGPUAttributes;
+  let Content = [{
+The number of work groups specifies the number of work groups when the kernel
+is dispatched.
+
+Clang supports the
+``__attribute__((amdgpu_num_work_groups(<x>, <y>, <z>)))`` attribute for the
```

Comment:
I think we prefer spelling it `[[clang::...]]` now in documentation.  I personally would prefer we move everyone to that as well.

Reply:
Added [[clang::...]] as suggested.